paper_scripts/training_scripts/finetune_personalized.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#./finetune_personalized.sh 0 /video-conf/scratch/vedatha.../.cpl /video-conf/scratch/vedantha/,,.cpk finetune
+gpu_index=$1
+base_checkpoint=$2
+netadapt_checkpoint=$3
+log_dir=$4
+
+# Run the command using the specified GPU index and file paths
+for person_id in "seth_meyers" "kayleigh" "needle_drop" "trevor_noah" "jen_psaki"
+do
```

Comment:
can you add sample commands to run both with and without depthwise and the top of the file